$MirOS: ports/emulators/qemu/snapshot/patches/patch-target-ppc_op_helper_c,v 1.1 2008/05/07 15:53:13 tg Exp $
$OpenBSD: patch-target-ppc_op_helper_c,v 1.5 2008/04/28 22:52:38 todd Exp $
--- target-ppc/op_helper.c.orig	Wed Sep 23 19:01:08 2009
+++ target-ppc/op_helper.c	Sat Oct 17 16:50:07 2009
@@ -16,6 +16,8 @@
  * You should have received a copy of the GNU Lesser General Public
  * License along with this library; if not, see <http://www.gnu.org/licenses/>.
  */
+#include <math.h>
+#include <stdio.h>
 #include <string.h>
 #include "exec.h"
 #include "host-utils.h"
